CMS-Enterprise/mint-app
# [MINT-3192] MTO Card in Collab Area

# MINT-3192

* Created MTOCard, test, and query, and i18n translations
* Added `mtoMatrix` mock to multiple tests
* Added MTOCard to Collaboration Area 
* Add 'model-to-operation' to `PageLockWrapper`


## Description



## How to test this change

1.  Navigate to Collaboration Area and find the MTO Card component and verify it matches figma
2. In React Developer Tools, multiple the `status` and ensure the status changes and button responds
3. Test locked state
4. Test recently edited state

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
